### ISSUE TITLE
Error in tree-from-tags

### ISSUE THREAD

**Issue by thomasluquet** [dbrock/bongo#14]
```
➜  bongo git:(master) ruby tree-from-tags.rb /home/zobi8225/Music 
Counting files... 720.
[0.00%] Processing `/home/zobi8225/Music/2013'...tree-from-tags.rb:213:in `rescue in block in <main>': uninitialized constant TagLib::BadFile (NameError)
    from tree-from-tags.rb:230:in `block in <main>'
    from /usr/lib/ruby/2.2.0/find.rb:48:in `block (2 levels) in find'
    from /usr/lib/ruby/2.2.0/find.rb:47:in `catch'
    from /usr/lib/ruby/2.2.0/find.rb:47:in `block in find'
    from /usr/lib/ruby/2.2.0/find.rb:42:in `each'
    from /usr/lib/ruby/2.2.0/find.rb:42:in `find'
    from tree-from-tags.rb:155:in `<main>'
```

I use this tablib-ruby : http://robinst.github.io/taglib-ruby/

**Comment by fm4d** [comment]
I have exactly this issue too.

**Comment by dbrock** [comment]
Possibly solved by #36?
